libraries/shared/src/BlendshapeConstants.cpp

```diff
@@ -76,3 +76,11 @@ const char* FACESHIFT_BLENDSHAPES[] = {
     "UserBlendshape9",
     ""
 };
+
+QMap<QString, int> blendshapeLookupMap = [] {
```

Comment:
Would it be strange if this were actually written in the constant casing like `BLENDSHAPE_LOOKUP_MAP` ? Maybe you could make it even a `const` ? since you assign to it only once right?